contracts/XVSVault/XVSVault.sol

```diff
@@ -911,6 +911,14 @@ contract XVSVault is XVSVaultStorage, ECDSA, AccessControlledV5, TimeManagerV5 {
         _initializeTimeManager(timeBased_, blocksPerYear_);
     }
 
+    /**
+     * @notice set Block per year in Time Manager
+     * @param blocksPerYear_ The number of blocks per year
+     */
+    function setBlocksPerYear(uint256 blocksPerYear_) external onlyAdmin {
```

Comment:
I think it would make sense if we integrate this new function with the ACM, granting the three timelock contracts to call it. We usually reserve the `onlyAdmin` modifier for changes in the contracts configured, not for the rest of parameters.

```suggestion
    function setBlocksPerYear(uint256 blocksPerYear_) external {
        _checkAccessAllowed("setBlocksPerYear(uint256)");
```